pkg/crosscluster/physical/stream_ingestion_frontier_processor.go

```diff
@@ -361,8 +361,16 @@ func (sf *streamIngestionFrontier) maybeUpdateProgress() error {
 		if replicatedTime.IsSet() && streamProgress.ReplicationStatus == jobspb.InitialScan {
 			streamProgress.ReplicationStatus = jobspb.Replicating
 			md.Progress.StatusMessage = streamProgress.ReplicationStatus.String()
-		} else if statusByStats != "" {
```

Comment:
could we deflake this test by simply removing this condition added in the og PR? I feel like adding new metrics is a separate unit of work than updating the status message of the PCR. Your og PR didn't add any e2e testing for metrics, which is fine. But, i think modifying status updates deserves it's own testing.

how about quickly deflake the test by removing this condition, then in a follow up, we think about how incorporate the status message updates into pcr?